be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -443,8 +455,9 @@ Status HdfsTextScanner::_create_or_reinit_reader() {
 Status HdfsTextScanner::_setup_io_ranges() const {
     if (_shared_buffered_input_stream != nullptr) {
         std::vector<io::SharedBufferedInputStream::IORange> ranges{};
-        for (int64_t offset = 0; offset < _scanner_params.file_size;) {
-            const int64_t remain_length = std::min(config::text_io_range_size, _scanner_params.file_size - offset);
+        const int64_t scan_range_end = _scanner_params.scan_range->offset + _scanner_params.scan_range->length;
```

Comment:
do we need to get min(file_size,scan_end) ?

Reply:
scan_range_end must <= file_size

Reply:
> scan_range_end must <= file_size

This is not a justifiable rationale. I'm afraid urgency alone does not justify compromising on quality assurance.
Besides, resolving inconsistent naming is quite simple,  it should only require a small time investment.